scripts/githooks/check-file-format.sh

```diff
@@ -7,7 +7,7 @@ set +e
 # according to the style defined in the `.editorconfig` file.
 #
 # Usage:
-#   $ ./check-file-format.sh
+#   $ check={all,staged-changes,working-tree-changes,branch} [dry_run=true] ./check-file-format.sh
 #
 # Options:
```

Comment:
I think now we can remove the `ALL_FILES` option from the doc as we replaced it by the `check` variable.